lib/js/util.mjs

```diff
@@ -113,3 +113,20 @@ export function validateOpenTypeTagString(tag) {
     }
     return [true, null];
 }
+
+// This class tells DOMTool to render HTML contents passed as a string.
+// It was created to allow passing labels containing HTML elements, such as
+// `span`s for icons loaded from the Material Symbols font.
+export class HTMLString extends String {}
```

Comment:
if `HTMLString` ges into `dom-tool.mjs`  I would put these functions into an own module under components, can be `icons.mjs`.

`icons.mjs` then would
```
import HTMLString from "../domTool.mjs"
```

Feels like the best approach to me. Unless I'm missing something.